### PR TITLE
hayper asked me to do this

### PR DESCRIPTION
**Pull request by vierofernando**
h [dont kill me](https://discord.com/channels/264445053596991498/264445053596991498/788253235168149524)

### CODE REVIEW

**Comment by xhayper** [comment]
h

**Review by alebot-dev — APPROVED** [review]
YESYWSYEYSYEYSYEYSYSYSYYEYSYEYSY3YSYEYSYEYSYDYSYEYSYEYSYEYSYESYESYWSYEYSYEYSYEYSYSYSYYEYSYEYSY3YSYEYSYEYSYDYSYEYSYEYSYEYSYES

File: .npmignore [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /packer.js
 /tsconfig.json
-/yarn.lock
 /.eslintrc.json
-/.idea/
```
Comment (Misly16) [inline .npmignore:5]: Why remove jetbrains files?

File: .npmignore [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+/yarn.lock
 /.gitignore
 /.all-contributorsrc
+/.idea/
 /node_modules/
 /test/
-.env
```
Comment (Misly16) [inline .npmignore:10]: Pushing .env files to npm is a bad idea

File: package.json [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
   "author": "ourcord",
   "license": "MIT",
-  "types": "src/websocket.d.ts",
+  "types": "dist/websocket.d.ts",
+  "runkitExampleFilename": "./examples/example.js",
   "scripts": {
-    "start": "node dist/example.index.js",
-    "upgrade": "node packager.js",
-    "prepublish": "tsc"
+    "prepare": "npm run compile",
```
Comment (Misly16) [inline package.json:11]: What does this script and the 2 below this one do?

**Review by Misly16 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (xhayper) [reply package.json:11]: Basiclly run script on spefic os,
We usually clean the output file before we compile it.

File: .npmignore (hunk 2 above)
Comment (xhayper) [reply .npmignore:10]: I didnt do this, bruh

File: .npmignore (hunk 1 above)
Comment (xhayper) [reply .npmignore:5]: I difnt

File: package.json (hunk 3 above)
Comment (Misly16) [reply package.json:11]: Although it may be better i dont really see the point of this, we compile the src into dist before publishing it to npm

Comment (xhayper) [reply package.json:11]: Can we talk on discord dm

Comment (Misly16) [reply package.json:11]: Sure

**Comment by alebot-dev** [comment]
h

**Pull request merged**